### PR TITLE
Removing POST data in GET request.

### PR DESCRIPTION
**Pull request by AlfZombie**
Removing POST data from HTTP GET method call for getting hub configuration. Removing the POST data as it is against HTTP standards. Tested without the payload and is working fine.

### CODE REVIEW

File: java/server/src/org/openqa/grid/internal/utils/SelfRegisteringRemote.java [hunk 1]
```diff
@@ -283,10 +283,6 @@ private JsonObject getHubConfiguration() throws Exception{
     String url = api.toExternalForm();
     BasicHttpEntityEnclosingRequest r = new BasicHttpEntityEnclosingRequest("GET", url);
 
-    JsonObject j = new JsonObject();
```
Comment (AlfZombie) [inline java/server/src/org/openqa/grid/internal/utils/SelfRegisteringRemote.java]: Ideally, there should be a check in HTTP object to throw exception if the method is GET and user tries to add payload (I've seen this check in OS source codes). Same can be added as a UT for the method that constructs the HTTP packet, which resides in apache code.

**Comment by andreastt** [comment]
@sevaseva?

**Comment by sevaseva** [comment]
@AlfZombie , have you signed the CLA http://goo.gl/qC50R ?
Also, can you please squash your commits.

**Comment by ddavison** [comment]
ping @AlfZombie .  Have you signed the CLA?

**Pull request closed without merging**